server/src/db/menu.js

```diff
@@ -0,0 +1,24 @@
+const menu = [
+  {
+    id: 1111,
+    name: "Fried Rice & chicken",
+    price: "$20"
+  },
+  {
+    id: 2222,
```

Comment:
There are multiple meals with the same ID. Don't you think this would cause a conflict when you try to perform an operation on just one item with id `2222`?

Reply:
In addition to this, you probably want to consider using an id generator like `uuid`.

Reply:
Okay boss...